spec/factories/tuks.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :tuk do
+    # to-do later
```

Comment:
NIT: (Read more about what a "nit" is, over [here](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.))

It's a common pattern to "to do"s styled as:

```
# TODO: {Name of the thing}
```

This is also the case for things that are broken or require attention, instead of writing:

```
# this is wierd, we need to delete null references
```

it's best to write it like:

```
# FIXME: This method breaks when the `user_id`` is `null`
```

## Other Tips

When writing comments, if there's anything related to "code", it's a good idea to write in backticks. For example:

```
## Good
# TODO: create a rake task for the books

## Better
# TODO: Create a Rake Task for the `Book`s
```